runtime/hydradx/src/evm/precompiles/multicurrency.rs

```diff
@@ -0,0 +1,250 @@
+//                    :                     $$\   $$\                 $$\                    $$$$$$$\  $$\   $$\
```

Comment:
this file is copied from acala as well?

Reply:
it is not copied, and i think the header is fine for this. Or what is the issue with it?

Reply:
nice